src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/PromotionTranslation.xml

```diff
@@ -24,6 +24,9 @@
             <itemOperation name="admin_get">
                 <attribute name="method">GET</attribute>
                 <attribute name="path">/admin/promotion-translations/{id}</attribute>
+                <attribute name="normalization_context">
+                    <attribute name="groups">admin:promotion:read</attribute>
+                </attribute>
```

Comment:
It should not be needed 🤔 Why did you configure it here?

Reply:
I have removed